### PR TITLE
Elemwise tester uint poc

### PR DESCRIPTION
**Pull request by benathi**
Let me know if this is along the line of what you're looking for. If I misunderstood, please let me know.

### CODE REVIEW

**Comment by nouiz** [comment]
that seem good. Do you have the time to check all tester in that file and update them if useful?

File: theano/tensor/tests/test_basic.py [hunk 1]
```diff
@@ -697,6 +699,9 @@ def check_floatX(inputs, rval):
         three_inputs_same_shapes=(rand(2, 3),
                                   rand(2, 3),
                                   rand(2, 3)),
+        three_inputs_same_shapes_uint = (randuint(2,3),
```
Comment (abergeron) [inline theano/tensor/tests/test_basic.py]: No spaces around the `=` since this is in a function call.

File: theano/tensor/tests/test_basic.py [hunk 2]
```diff
@@ -545,6 +545,8 @@ def rand_nonzero(shape, eps=3e-4):
 def randint(*shape):
     return numpy.random.random_integers(-5, 5, shape)
 
+def randuint(*shape):
+    return numpy.array(numpy.random.randint(5, size=shape) , dtype=numpy.uint32)
```
Comment (abergeron) [inline theano/tensor/tests/test_basic.py]: No space before a comma.

**Comment by nouiz** [comment]
I closed to trigger a new travis build. There was a a flake8 violation in the trunk that is now fixed. When travis pass, we can merge.

**Pull request merged**